templates/edit_movies_form.html

```diff
@@ -1,11 +1,8 @@
-{% extends '_layout.html' %} 
-
-{% block title %}Movie Rating App | Edit Movie{%endblock %} 
-
-{% block body %}
+{% extends '_layout.html' %} {% block title %}Movie Rating App | Edit
+Movie{%endblock %} {% block body %}
 <h1 class="mb-5"></h1>
 <!-- TODO: Feature 5 -->
-<form method="POST" action="/movies/{{movie.movie_id}}">
+<form method="post" action="/movies/{{movie.movie_id}}">
   <!-- Title of movie -->
   <p>
```

Comment:
I believe the p tags surrounding the labels and inputs aren't necessary. Other than that, everything seems to be looking good. You could also add bootstrap classes in the inputs and labels like form-control and form-label.

Reply:
true that, maybe next time